### DISCUSSION TITLE
The utility contains an invalid theme value and was not generated warning is causing Next to fail to build the project

### DISCUSSION THREAD

**Discussion by matagaralph** [tailwindlabs/headlessui#3568] [Help] [answered]
I'm getting this warning but the project is the project fails to compile. 
`The utility `data-[slot=avatar]:*:[--avatar-radius:theme(borderRadius.DEFAULT)]` contains an invalid theme value and was not generated.
Failed to compile.`

Stack Trace
/node_modules/.pnpm/@headlessui+react@1.7.19_react-dom@19.0.0-rc.1_react@19.0.0-rc.1__react@19.0.0-rc.1/node_modules/@headlessui/react/dist/utils/stable-collection.js
Attempted import error: '__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED' is not exported from 'react' (imported as 'r').

Import trace for requested module:
./node_modules/.pnpm/@headlessui+react@1.7.19_react-dom@19.0.0-rc.1_react@19.0.0-rc.1__react@19.0.0-rc.1/node_modules/@headlessui/react/dist/utils/stable-collection.js
./node_modules/.pnpm/@headlessui+react@1.7.19_react-dom@19.0.0-rc.1_react@19.0.0-rc.1__react@19.0.0-rc.1/node_modules/@headlessui/react/dist/components/tabs/tabs.js

I'm using Next 15 can you help what could be the issue here

**Comment by kms0219kms** [comment, marked as answer]
#3167 Solved from this

**Reply by matagaralph** [reply]
> #3167 Solved from this

Thanks buddy.
